common/utils/index.js

```diff
@@ -550,19 +550,31 @@ function hasChangesInForbiddenSections(diff) {
   return false;
 }
 
-function unifyDiffResult(result) {
+function unifyDiffResult(result, ignoreTags) {
```

Comment:
@jayeshjk Can you please write one sample diff result in comment above this method. It is really confusing and difficult to understand w/o it.